data/sles4sap/qe_sap_deployment/qesap_aws.yaml

```diff
@@ -36,6 +36,7 @@ ansible:
     secondary_site: 'miky'
   create:
     - registration.yaml -e reg_code='%SCC_REGCODE_SLES4SAP%' -e email_address=testing@suse.com
+    - fully-patch-system.yaml
```

Comment:
I'm not sure we want all qesap regression to use this

Reply:
Not sure too.

Reply:
I'd say either remove it or leave it depending on what would improve the stability of the qesap regression tests. Removing the `fully-patch-system.yaml` would result in a quicker test, but if by some change the unpatched system is broken and a `zypper patch` would fix the issue then it would be preferable. I'm somewhat leaning to removing it, but I'm not 100% sure.

Reply:
> I'd say either remove it or leave it depending on what would improve the stability of the qesap regression tests. Removing the `fully-patch-system.yaml` would result in a quicker test, but if by some change the unpatched system is broken and a `zypper patch` would fix the issue then it would be preferable. I'm somewhat leaning to removing it, but I'm not 100% sure.

I like to have a place (test) to check if the full patch makes any difference. I like to have only one test using it. Or one test not using it.

Reply:
Removed `fully-patch-system.yaml` from `data/sles4sap/qe_sap_deployment/qesap_aws.yaml`.